src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -1,11 +1,38 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.OneToOne;
+import java.util.ArrayList;
+import java.util.List;
 
-@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Entity(name = "carts")
 public class Cart {
 
     @Id
+    @GeneratedValue
     private long id;
+
+    @ManyToMany
+    @JoinTable(
```

Comment:
podałem ci to co masz podmienić w `<>` w związku z czym tych nawiasów ma nie być